.github/workflows/label_merge_conflicts.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Merge Conflict Detection'
+on:
+  push:
+    branches:
+      - master
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mschilde/auto-label-merge-conflicts@master
+        with:
+          CONFLICT_LABEL_NAME: 'Merge Conflict'
```

Comment:
This was modified from the original to make sure its accurate for our label, and a new label isnt made when this runs